extras/qibosoq_paper_runcards/scaling_benchmarks.yml

```diff
@@ -0,0 +1,292 @@
+# executed with `qq auto scaling_benchmarks.yml --no-update -o output_folder`
+# all the different sweepers were executed separately
```

Comment:
I know that there is the common habit of uncommenting a piece at a time and run, but this is a bit confusing.

I'd rather suggest splitting this runcard in the 6 sections is consisting of, and keep only descriptive comments (and no commented code at all).

Reply:
If you want to group them together, a proper way would be a subfolder (but it is not required, choose your favorite way).

Reply:
With --no-update it should work even in the same runcard. I removed all the comments and it should now work fine